tests/test_sync.py

```diff
@@ -9,6 +9,9 @@
 from base import DirSyncTests
 import trees
 
+import shutil
+import stat
+from pathlib import Path
```

Comment:
I have an OCD on the import statements order ;)
Would you mind moving them up just under `from time import sleep`?
I generally group all built-in imports at the top, then another group with packages and finally then local modules.